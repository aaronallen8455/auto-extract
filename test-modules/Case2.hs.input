module Case2 where

test x = 1 + EXTRACT@simple (x + 1)
