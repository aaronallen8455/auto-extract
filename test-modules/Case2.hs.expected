module Case2 where

test x = 1 + simple x

simple :: Num a => a -> a
simple x = x + 1
